native_client/client.cc

```diff
@@ -63,15 +63,15 @@ int
 main(int argc, char **argv)
 {
   if (argc < 4 || argc > 7) {
-    printf("Usage: deepspeech MODEL_PATH AUDIO_PATH ALPHABET_PATH [LM_PATH] [TRIE_PATH] [-t]\n");
+    printf("Usage: deepspeech [MODEL_PATH] [ALPHABET_PATH] [LM_PATH] [TRIE_PATH] [AUDIO_PATH] [-t]\n");
```

Comment:
Model, alphabet and audio are not optional arguments, so you should not enclose them in brackets.